Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialFunctions/EvaluateStandardSurface.azsli

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Features/PBR/Surfaces/StandardSurface.azsli>
+#include <Atom/Features/MatrixUtility.azsli>
+
+void EvaluateStandardSurface(
```

Comment:
Same, unclear why this was factored out from the only file in which it's used.

Reply:
https://github.com/o3de/o3de/pull/7437#discussion_r801076507